src/polyfills.js

```diff
@@ -0,0 +1 @@
+import 'context-filter-polyfill';
```

Comment:
why did you have to separate this in another file?

Reply:
At the time I thought it was a good idea to have a dedicated file in case we want to add other polyfills to the project for any reason. I'm fine with removing it and importing it directly where we need it though!

Reply:
for now it just looks like an extra file. lets just put it directly

Reply:
Sounds good, will make the changes right away